independent-projects/arc/runtime/src/main/java/io/quarkus/arc/InjectableContext.java

```diff
@@ -61,5 +60,13 @@ default boolean isNormal() {
          */
         Map<InjectableBean<?>, Object> getContextualInstances();
 
+        /**
+         * 
+         * @return {@code true} if there are no contextual instances stored in the context
+         */
+        default boolean isEmpty() {
```

Comment:
This util method (and its overriden version in req. context impl) is not used anywhere, do you still want to keep it?

Reply:
Nope. Let's remove it!

Reply:
Done.